custom_components/waste_collection_schedule/waste_collection_schedule/source/dillingen_saar_de.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+import requests
+from waste_collection_schedule import Collection
+from waste_collection_schedule.service.ICS import ICS
+from urllib.parse import quote
+from datetime import datetime
+
+
+
+TITLE = "Dillingen Saar" # Title will show up in README.md and info.md
+DESCRIPTION = "Source script for waste collection Dillingen Saar"  # Describe your source
+URL = "https://www.dillingen-saar.de/"  # Insert url to service homepage. URL will show up in README.md and info.md
+TEST_CASES = {  # Insert arguments for test cases to be used by test_sources.py script
+    "Am Fischerberg": {"street": "Am Fischerberg"}, #https://service-dillingen-saar.fbo.de/date/Am%20Fischerberg/2023-01-01/+1%20year/?format=ics&type=rm,gs,bio,pa
+    "Odilienplatz": {"street": "Odilienplatz"},     #https://service-dillingen-saar.fbo.de/date/Odilienplatz/2023-01-01/+1%20year/?format=ics&type=rm,gs,bio,pa
+    "Lilienstraße": {"street": "Lilienstraße"},      #https://service-dillingen-saar.fbo.de/date/Lilienstra%C3%9Fe/2023-01-01/+1%20year/?format=ics&type=rm,gs,bio,pa
+    "Joseph-Roederer-Straße": {"street": "Joseph-Roederer-Straße"} #https://service-dillingen-saar.fbo.de/date/Joseph-Roederer-Stra%C3%9Fe/2023-01-01/+1%20year/?format=ics&type=rm,gs,bio,pa
+}
+
+API_URL = "https://service-dillingen-saar.fbo.de/date/"
+REMINDER_DAY = (
```

Comment:
obsolete, please remove

Reply:
same here. Running into following error if running the test script:
`➜  test git:(master) ✗ python test_sources.py -s dillingen_saar_de
Testing source dillingen_saar_de ...
Traceback (most recent call last):
  File "/home/plex/hacs_waste_collection_schedule/custom_components/waste_collection_schedule/waste_collection_schedule/test/test_sources.py", line 153, in <module>
    main()
  File "/home/plex/hacs_waste_collection_schedule/custom_components/waste_collection_schedule/waste_collection_schedule/test/test_sources.py", line 71, in main
    assert "TITLE" in names
AssertionError
`

Reply:
Same here, I just meant REMINDER_DAY.